[loxone] LxControlIRoomControllerV2Test fails on Windows

Build of Loxone binding is failing due to a failing test:
```
$ mvn clean install
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building openHAB Add-ons :: Bundles :: Loxone Binding 2.5.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- maven-clean-plugin:3.0.0:clean (default-clean) @ org.openhab.binding.loxone ---
[INFO] Deleting D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\target
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (enforce-java) @ org.openhab.binding.loxone ---
[INFO]
[INFO] --- directory-maven-plugin:0.3.1:directory-of (directories) @ org.openhab.binding.loxone ---
[INFO] Directory of org.openhab.addons:org.openhab.addons.reactor set to: D:\dev\openhab2\git\openhab2-addons
[INFO]
[INFO] --- maven-dependency-plugin:3.1.1:unpack-dependencies (embed-dependencies) @ org.openhab.binding.loxone ---
[INFO]
[INFO] --- karaf-maven-plugin:4.2.6:features-generate-descriptor (compile) @ org.openhab.binding.loxone ---
[INFO] Generating feature descriptor file D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\target\feature\feature.xml
[INFO]  Tree listing is saved here: D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\target\history\treeListing.txt

[INFO] ...done!
[INFO] Attaching features XML
[INFO]
[INFO] --- maven-resources-plugin:3.0.2:resources (default-resources) @ org.openhab.binding.loxone ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 2 resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.0:compile (default-compile) @ org.openhab.binding.loxone ---
[INFO] Changes detected - recompiling the module!
[INFO] D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\src\main\java\org\openhab\binding\loxone\internal\LxServerHandler.java:[540,21315] Unsafe interpretation of method return type as '@NonNull' based on substitution 'T=LxResponse.@NonNull LxResponseCfgApi'. Declaring type 'Gson' doesn't seem to be designed with null type annotations in mind
[INFO] D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\src\main\java\org\openhab\binding\loxone\internal\LxDynamicStateDescriptionProvider.java:[75,2924] Unsafe interpretation of method return type as '@NonNull' based on the receiver type '@NonNull Map<@NonNull ChannelUID,@NonNull StateDescription>'. Type 'Map<K,V>' doesn't seem to be designed with null type annotations in mind
[WARNING] D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\src\main\java\org\openhab\binding\loxone\internal\types\LxTags.java:[18,510] The import org.openhab.binding.loxone.internal.controls.LxControl is never used
[INFO] D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\src\main\java\org\openhab\binding\loxone\internal\LxWebSocket.java:[293,12108] Unsafe interpretation of method return type as '@NonNull' based on substitution 'T=@NonNull LxConfig'. Declaring type 'Gson' doesn't seem to be designed with null type annotations in mind
[INFO] D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\src\main\java\org\openhab\binding\loxone\internal\LxWebSocket.java:[318,12930] Unsafe interpretation of method return type as '@NonNull' based on substitution 'T=@NonNull LxResponse'. Declaring type 'Gson' doesn't seem to be designed with null type annotations in mind
[WARNING] D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\src\main\java\org\openhab\binding\loxone\internal\types\LxConfig.java:[68,2409] Null type mismatch (type annotations): 'null' is not compatible to the free type variable 'T'
[WARNING] D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\src\main\java\org\openhab\binding\loxone\internal\types\LxCategory.java:[15,454] The import org.openhab.binding.loxone.internal.controls.LxControl is never used
[WARNING] D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\src\main\java\org\openhab\binding\loxone\internal\security\LxWsSecurity.java:[27,895] The import org.openhab.binding.loxone.internal.LxServerHandler is never used
[WARNING] D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\src\main\java\org\openhab\binding\loxone\internal\types\LxResponse.java:[132,4015] Null type mismatch (type annotations): 'null' is not compatible to the free type variable 'T'
[WARNING] D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\src\main\java\org\openhab\binding\loxone\internal\LxServerHandlerApi.java:[19,555] The import org.eclipse.smarthome.core.thing.Thing is never used
[WARNING] D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\src\main\java\org\openhab\binding\loxone\internal\LxServerHandlerApi.java:[24,825] The import org.openhab.binding.loxone.internal.security.LxWsSecurity is never used
[WARNING] D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\src\main\java\org\openhab\binding\loxone\internal\controls\LxControlIRoomControllerV2.java:[76,3417] The value of the field LxControlIRoomControllerV2.logger is not used
[INFO]
[INFO] --- bnd-maven-plugin:4.2.0:bnd-process (default) @ org.openhab.binding.loxone ---
[WARNING] D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\pom.xml [0:0]: Unused Export-Package instructions: [org.openhab.*]
[WARNING] D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\pom.xml [0:0]: Unused Import-Package instructions: [io.swagger.annotations.*, javax.annotation.security.*, org.eclipse.jdt.annotation.*, org.openhab.core.automation.annotation.*, org.openhab.*, com.google.common.*]
[INFO]
[INFO] --- maven-resources-plugin:3.0.2:testResources (default-testResources) @ org.openhab.binding.loxone ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 3 resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.0:testCompile (default-testCompile) @ org.openhab.binding.loxone ---
[INFO] Changes detected - recompiling the module!
[INFO] D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\src\test\java\org\openhab\binding\loxone\internal\controls\LxServerHandlerDummy.java:[74,2595] Unsafe interpretation of method return type as '@NonNull' based on substitution 'T=@NonNull LxConfig'. Declaring type 'Gson' doesn't seem to be designed with null type annotations in mind
[INFO]
[INFO] --- maven-surefire-plugin:2.22.1:test (default-test) @ org.openhab.binding.loxone ---
[INFO]
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlAlarmNoPresenceTest
[INFO] Tests run: 9, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.406 s - in org.openhab.binding.loxone.internal.controls.LxControlAlarmNoPresenceTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlAlarmWithPresenceTest
[INFO] Tests run: 11, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.031 s - in org.openhab.binding.loxone.internal.controls.LxControlAlarmWithPresenceTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlDimmerTest
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlDimmerTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlInfoOnlyAnalogTempTagTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlInfoOnlyAnalogTempTagTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlInfoOnlyAnalogTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlInfoOnlyAnalogTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlInfoOnlyDigitalTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlInfoOnlyDigitalTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlIRoomControllerV2Test
[ERROR] Tests run: 7, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.063 s <<< FAILURE! - in org.openhab.binding.loxone.internal.controls.LxControlIRoomControllerV2Test
[ERROR] testChannels(org.openhab.binding.loxone.internal.controls.LxControlIRoomControllerV2Test)  Time elapsed: 0 s  <<< FAILURE!
org.junit.ComparisonFailure: expected:<%.1f[]▒> but was:<%.1f[▒]▒>
        at org.openhab.binding.loxone.internal.controls.LxControlIRoomControllerV2Test.testChannels(LxControlIRoomControllerV2Test.java:63)

[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlJalousieTest
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlJalousieTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlLeftRightAnalogTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlLeftRightAnalogTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlLeftRightDigitalTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlLeftRightDigitalTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlLightControllerV2Test
[INFO] Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlLightControllerV2Test
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlMeterTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlMeterTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlPushbuttonTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlPushbuttonTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlRadioTest
[INFO] Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlRadioTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlSliderTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlSliderTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlSwitchTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlSwitchTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlTextStateTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlTextStateTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlTimedSwitchTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlTimedSwitchTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlTrackerTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlTrackerTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlUpDownAnalogTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlUpDownAnalogTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlUpDownDigitalTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlUpDownDigitalTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlValueSelectorIncrTest
[INFO] Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlValueSelectorIncrTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlValueSelectorTest
[INFO] Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlValueSelectorTest
[INFO] Running org.openhab.binding.loxone.internal.controls.LxControlWebPageTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.loxone.internal.controls.LxControlWebPageTest
[INFO]
[INFO] Results:
[INFO]
[ERROR] Failures:
[ERROR]   LxControlIRoomControllerV2Test.testChannels:63->LxControlTest.testChannel:126->LxControlTest.testChannel:97 expected:<%.1f[]▒> but was:<%.1f[▒]▒>
[INFO]
[ERROR] Tests run: 114, Failures: 1, Errors: 0, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 8.953 s
[INFO] Finished at: 2019-10-23T10:44:34+02:00
[INFO] Final Memory: 32M/679M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.1:test (default-test) on project org.openhab.binding.loxone: There are test failures.
[ERROR]
[ERROR] Please refer to D:\dev\openhab2\git\openhab2-addons\bundles\org.openhab.binding.loxone\target\surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
```

The brackets are added by JUnit to show where the mismatch is. For me it shows the following error on Windows:

`[ERROR]   LxControlIRoomControllerV2Test.testChannels:63->LxControlTest.testChannel:126->LxControlTest.testChannel:97 expected:<%.1f[]°> but was:<%.1f[Â]°>`

Maybe something went wrong while copy/pasting the unicode in @lolodomo's comment.

--- a/bundles/org.openhab.binding.loxone/src/test/java/org/openhab/binding/loxone/internal/controls/LxServerHandlerDummy.java
+++ b/bundles/org.openhab.binding.loxone/src/test/java/org/openhab/binding/loxone/internal/controls/LxServerHandlerDummy.java
@@ -18,6 +18,7 @@ import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
+import java.nio.charset.Charset;
 import java.util.HashMap;
 import java.util.LinkedList;
 import java.util.Map;
@@ -64,7 +65,7 @@ public class LxServerHandlerDummy implements LxServerHandlerApi {
     void loadConfiguration() {
         InputStream stream = LxServerHandlerDummy.class.getResourceAsStream("LoxAPP3.json");
         assertNotNull(stream);
-        BufferedReader reader = new BufferedReader(new InputStreamReader(stream));
+        BufferedReader reader = new BufferedReader(new InputStreamReader(stream, Charset.forName("UTF-8")));
         assertNotNull(reader);
         String msg = reader.lines().collect(Collectors.joining(System.lineSeparator()));
         assertNotNull(msg);

